tests/stage/one/test_s9_if.py

```diff
@@ -0,0 +1,11 @@
+
+import json
+
+
+async def test_if(test_cli):
+    req_data = json.dumps({'field_1': 'x>18'})
+    res = await test_cli.post('/stage/one/if', data=req_data)
+    assert res.status == 200
+    res_data = await res.json()
+    assert res_data['data']['result'] is True
+    await test_cli.close()
```

Comment:
Please read the new [CONTRIBUTING.md](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md#provide-a-test) to fit new test code.